.zuul.yaml

```diff
@@ -163,7 +163,7 @@
     name: infrawatch/feature-verification-tests
     periodic:
       jobs:
-        - telemetry-openstack-meta-content-provider-master:
+        - openstack-k8s-operators-content-provider:
             override-checkout: main
         - functional-periodic-telemetry-with-ceph
```

Comment:
The job definition includes a dependency on the meta content provider. The job variant that runs in the periodic line needs to have the dependency changed so it depends on the openstack-k8s-operators-content-provider.